src/bindings/python/wheel/setup.py

```diff
@@ -422,7 +422,7 @@ def copy_package_libs(self, src_dirs):
         for src_dir in src_dirs:
             # additional blacklist filter, just to fix cmake install issues
             blacklist_patterns = [  # static libraries and PBD files
-                                    "^.*\\.a$", "^.*\\.lib$", "^.*\\.pdb$",
```

Comment:
what is the difference in binary size of wheel in this case?

Reply:
~1mb more